src/test/java/nextstep/ladder/domain/LineTest.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.ladder.domain;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LineTest {
+
+    @Test
+    void 입력된_숫자_만큼_라인이_생성_된다() {
```

Comment:
```suggestion
    @Test
    @DisplayName("입력된 숫자 만큼 라인이 생성 된다")
    void test() {
```

개인적으로는 `@DisplayName` 이 있으니 테스트에 굳이 한글 메소드명을 사용할 필요는 없다고 생각합니다.
코틀린과는 다르게 자바에선 메소드명 제약도 강하니까요.

`@DisplayName` 은 띄워쓰기나 개행문자도 표현할 수 있는 등 장점이 많습니다.